src/lib.rs

```diff
@@ -619,6 +622,10 @@ impl Uuid {
     ///
     /// assert_eq!(expected_uuid, uuid);
     /// ```
+    #[deprecated(
+        since = "0.7.2",
+        note = "please use the `UuidBuilder` instead"
```

Comment:
`note = "use `uuid::builder::UuidBuilder` instead"`

Reply:
Something about `uuid::builder::UuidBuilder` doesn't sit right with me... We could just go with `uuid::Builder`?

Reply:
yeah its not like we are going to have several builders so `uuid::Builder` should be  fine

Reply:
So should I reexport the `Builder` via `pub use builder::Builder;` in the `lib.rs` as well as reexporting it through the prelude?